content/concepts/payment-system-basics/ledgers.md

```diff
@@ -23,10 +23,28 @@ As its name might suggest, a ledger's state tree is a tree data structure. Each
 
 In the case of transactions, the identifying hash is based on the signed transaction instructions, but the contents of the transaction object when you look it up also contain the results and metadata of the transaction, which are not taken into account when generating the hash.
 
+## Open, Closed, and Validated Ledgers
+
+The `rippled` server makes a distinction between ledger versions that are _open_, _closed_, and _validated_. A server has one open ledger, a history of immutable, validated ledgers, and any number of closed but unvalidated ledgers. The following table summarizes the difference:
+
+| Ledger Type                      | Open                        | Closed                                     | Validated |
```

Comment:
Looking at the rendered version of this table, the "Ledger Type" cell is a little confusing, and runs into a "is this a header for the row of the column?" problem. I think it can be improved by some combination of:
1. Put a colon at the end `Ledger Type:`/
2. Right justify the text so it visually lines up with the type names.
3. Remove it entirely.

Reply:
Went with adding a colon.

Re: option 3, [accessibility guidelines for tables](https://webaim.org/techniques/tables/data) say:

> Table headers should never be empty. This is particularly of concern for the top-left cell of some tables

Reply:
It's a small thing, but I think it helps. Thanks.